packages/react-scripts/config/webpack.widget.config.js

```diff
@@ -150,7 +150,7 @@ module.exports = function(webpackEnv) {
     entry: paths.appIndexWidgetJs,
     output: {
       libraryTarget: 'umd',
-      library: 'additiveeu_voucher',
+      library: appPackageJson.widget,
```

Comment:
Lei damit es jetz versteats.
Ob jetz isch jedes Widget wos nui gmocht wert mit cra, als erstes werden donn stot die react-script die additive-react-scripts gnummen.

Und in dor package json kimp nor no dor name vom build eini, beim voucehr wars donn `widget: 'additiveeu_voucher'` dor build nimp des nor her und nimp sel als global scope name.


Sel brauchts donn beim widget wenns in a seite eingebunden wert, segs donn im pr beim widget wia des verwendet wert